### DISCUSSION TITLE
Request for guidance to participate in the Octernships program.

### DISCUSSION THREAD

**Discussion by ACSG-64** [community/community#73107] [GitHub Education]
### Hi everyone,

Good morning everyone.

I am opening this post because I would like to participate in the Octernships program and I think I meet the requirements but on the page it appears that I am not eligible. 
![image](https://github.com/community/community/assets/50815104/1c610e6f-e4b0-4a16-980b-54a343ac3156)

I am Colombian, I live in Colombia (one of the eligible countries), I am legally able to work and my GitHub account is verified in the GitHub Education program since I am currently a university student.

I would appreciate if you could guide me on the steps to follow in order to participate in this program.

Thanks in advance.

**Comment by williamregal** [comment]
The [HP HPE6-A85 November 2023 Release](https://www.valid4sure.com/HPE6-A85-exam-answers.html), which are frequently created by qualified people with experience in their field, are an important tool for exam preparation. These study materials are a set of questions and answers that closely resemble the structure and subject matter of the real exam. They are made to check students' knowledge, acquaint them with the format of the exam, and point out any areas that need more research.
